package.json

```diff
@@ -10,8 +10,8 @@
     "lint:fix": "eslint --fix --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "lint": "eslint --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "format": "npx prettier --write .",
-    "pretest": "rimraf ./.nyc_output",
```

Comment:
Removing this script because the removal of the `.nyc_output` directory was causing issues when trying to run the `nyc` cli utility. Specifically, it couldn't find the directory. During my testing, the existence of this directory was never an issue and it was never documented as a recommended installation step.